src/screens/WebsocketResearchScreen/SampleChatInfoScreen.tsx

```diff
@@ -173,15 +226,39 @@ const SampleChatInfoScreen = () => {
     );
   };
 
-  const {anon_user_info_color_code, anon_user_info_emoji_code, anon_user_info_emoji_name} =
-    channelInfo?.rawJson?.channel || {};
+  const renderImageComponent = (item) => {
+    if (!isContainUrl(item?.user?.image) || item?.user?.name === ANONYMOUS_USER) {
+      return (
+        <View style={styles.mr7}>
+          <AnonymousIcon
+            color={anon_user_info_color_code}
+            emojiCode={anon_user_info_emoji_code}
+            size={normalize(48)}
+          />
+        </View>
+      );
+    }
+    return (
+      <View style={styles.mr7}>
+        <FastImage style={styles.imageUser} source={{uri: item?.user?.image}} />
+      </View>
+    );
+  };
+
+  const countParticipat = () => {
+    return `(${channelInfo?.members?.length})`;
+  };
 
   return (
     <SafeAreaView style={styles.container}>
       <StatusBar translucent={false} />
-      {isLoadingMembers ? null : (
+      {isLoadingMembers || loadingChannelInfo ? null : (
         <>
-          <AnonymousChatInfoHeader isCenter onPress={goBack} title={channelInfo?.name} />
+          <AnonymousChatInfoHeader
+            isCenter
+            onPress={goBack}
+            title={`${getChatName(channelInfo?.name, profile?.myProfile?.username)}`}
+          />
           <View style={styles.lineTop} />
           <ScrollView nestedScrollEnabled={true}>
             <SafeAreaView>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [229-299]

The rendering of participants in the chat has been updated to use a new function `renderImageComponent` and to display the count of participants using a new function `countParticipat`. Ensure that these changes work as expected and handle all edge cases.



<!-- This is an auto-generated comment by OSS CodeRabbit -->